src/main/java/page_objects/TimetablePage.java

```diff
@@ -0,0 +1,19 @@
+package page_objects;
+
+import elements.Link;
+import helpers.ElementHelper;
+import org.openqa.selenium.By;
+
+public class TimetablePage extends BasePage {
+    private String xpathForBookTicket = " //td[count(//th[text()='Depart Station']/preceding-sibling::th)+1][text()='%s']"
```

Comment:
xpath like this is really hard to maintain and debug if something went wrong.
my suggestion is to write a function to find the needed index. then use it in a simpler xpath.
but this is not an easy thing to do. so it's okay if you can't do that.